include/vcpkg/base/messages.h

```diff
@@ -2091,5 +2091,168 @@ namespace vcpkg
                     (msg::path),
                     "",
                     "Failed to parse {path}, expected a top-level object.");
-
-}
+    DECLARE_MESSAGE(FailedToParseSerializedManifest,
```

Comment:
Should this also reuse `InternalErrorMessageContact` ?

Reply:
InternalErrorMessageContact doesn't take a path as an argument.